### PR TITLE
unify readme and in app instructions with Discord addition

### PR DESCRIPTION
**Pull request by mwickett**
Based on feedback, this unifies and duplicates (mostly, doesn't include the Button component and the Figma embed) instructions across the `readme` file and in-app instructions. 

It also adds a section on using the Discord channel.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,79 @@
...
+
+We expect this to be done within a day or two's worth of full-time work - about 8 - 14 hours. We know that you likely have other commitments, and so we generally look at about week for candidates to complete the challenge. If you're approaching 14 hours of total time spent, just stop where you are and we'll complete the evaluation based on what you have done.
+
+### Using the Discord channel
+
+You will have been added to a private discord channel with a few members of our team. The purpose of this channel is to give a space for us to collaborate together, again, to simulate us working together. Use the channel to ask questions, get clarity on anything ambiguous and share your approach to the work. Asking questions and being active in the channel are positive things, not negative. We don't expect you to have all the answers! There aren't any tricks in this challenge, but just like in real work, there are areas of ambiguity so if you're completing the challenge without any questions, you should reconsider.
+
```
Comment (mwickett) [inline README.md:24]: This is a net new addition.

**Review by kaitlynnefuery — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (dstaley) [reply README.md:24]: > you should reconsider.

Could we be more specific here? Maybe "reconsider your assumptions and collaborate with the team to ensure the best possible outcome" or something to that effect?

Comment (RubenSandwich) [reply README.md:24]: Putting that wording in this PR, before merging.

**Comment by mwickett** [comment]
thank you!

**Pull request merged**